packages/emails/email-manager.ts

```diff
@@ -43,19 +48,37 @@ export const sendScheduledEmails = async (calEvent: CalendarEvent) => {
   emailsToSend.push(
     new Promise((resolve, reject) => {
       try {
-        const scheduledEmail = new OrganizerScheduledEmail(calEvent);
+        const scheduledEmail = new OrganizerScheduledEmail(clonedEvent);
         resolve(scheduledEmail.sendEmail());
       } catch (e) {
         reject(console.error("OrganizerScheduledEmail.sendEmail failed", e));
       }
     })
   );
 
+  if (clonedEvent.team) {
+    for (const teamMember of clonedEvent.team.members) {
+      emailsToSend.push(
+        new Promise((resolve, reject) => {
+          try {
+            const scheduledEmail = new OrganizerScheduledEmail(clonedEvent, undefined, teamMember);
```

Comment:
By passing a team member, we can extract their locale and translation for the email